class.jetpack-plan.php

```diff
@@ -167,6 +168,7 @@ public static function get() {
 
 		if ( in_array( $plan['product_slug'], $business_plans, true ) ) {
 			$supports[]    = 'akismet';
+			$supports[]    = 'opentable';
```

Comment:
```suggestion
```

Reply:
I've been meaning to do that, thanks for spotting it for me!